packages/testing/src/steps/generic.steps.ts

```diff
@@ -58,6 +69,45 @@ export function setupGenericSteps() {
     }
   );
 
+  When(
```

Comment:
ok, this has ceased to be generic testing functions - I think there is a way to get back a promise and then wait for it, and then see the promise error.  So, separate steps but it achieves the same thing

Reply:
I'll move these into getAgent. The issue is not the result vs. error, it was passing the instrument in. It't not actually correct here (arguments are missing), which I thought I'd fixed but maybe haven't pushed?

Reply:
done